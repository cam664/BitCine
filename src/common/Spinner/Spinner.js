import spinner from '../../assets/img/spinner.gif';

/** @jsx jsx */
import { css, jsx } from '@emotion/core';
import styles from './SpinnerStyles';

const Spinner = () => <img css={css`${styles.center}`} src={spinner} />

export default Spinner;