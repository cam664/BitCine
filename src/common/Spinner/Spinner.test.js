import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './Spinner';

describe('Spinner', () => {
  it('renders a snapshot', () => {
    const component = renderer.create(
      <Spinner />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});