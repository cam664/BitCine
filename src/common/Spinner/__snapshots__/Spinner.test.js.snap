// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders a snapshot 1`] = `
<img
  className="css-14hden7-Spinner"
  src="spinner.gif"
/>
`;
